configure

```diff
@@ -892,7 +892,7 @@ Description: LibRHash shared library
 Version: ${RHASH_VERSION}
 Cflags: -I\${includedir}
 Libs: -L\${libdir} -lrhash
-Libs.private: ${OPENSSL_LDFLAGS}
+Libs.private: ${OPENSSL_LDFLAGS} ${GETTEXT_LDFLAGS} ${LIBDL_LDFLAGS}
```

Comment:
1.  OPENSSL_LDFLAGS already include $LIBDL_LDFLAGS,
when OpenSSL is onfigure for runtime loading. And this is the only case, when we need libdl.
2. GETTEXT_LDFLAGS are nor used, nor linked by LibRHash. It's used by RHash binary.

If you got a compilation error, please, provide it.

BTW, I've added generation only of _librhash.pc_, because nobody should link with RHash binary, only with the library.